src/app/actions.js

```diff
@@ -57,3 +58,13 @@ export function logOut() {
     dispatch(clearUserData());
   };
 }
+
+/**
+  Toggle collapsible sidebar
+*/
+export function collapsibleSidebar(toggle) {
```

Comment:
In an effort to support the 'pin-able' sidebar you could convert this to accept an enum. That way it could be set to various states like... `['closed', 'open', 'floating', 'pushing']`